### DISCUSSION TITLE
What`s generally considered visually lossless command line for this fork?

### DISCUSSION THREAD

**Discussion by necros2k7** [gianni-rosato/svt-av1-psy#107] [Q&A]
@1480c1 . @hguermaz , @joelsole As of now I use ffmpeg -c:v libsvtav1 -crf 34  (I know it`s still subjective by many)

**Comment by gianni-rosato** [comment]
There isn't a specific CRF value that will *always* be visually lossless. I usually recommend starting at CRF 29 at 1080p and going from there though

**Reply by necros2k7** [reply]
> There isn't a specific CRF value that will _always_ be visually lossless. I usually recommend starting at CRF 29 at 1080p and going from there though
I meant not _always_ but on _average_. So your bet is on 29 right? Any additional flags for command line above?

**Reply by gianni-rosato** [reply]
That average would have such a high standard deviation that it wouldn't be informative at all. My recommendation is to inspect your source and decide what to do based on some testing. My testing usually begins with CRF 29.
